README.md

```diff
@@ -393,53 +439,62 @@ separate flags are separated by spaces, e.g.:
 
 
 <colgroup>
-<col  class="left" />
+<col  class="org-left" />
 
-<col  class="right" />
+<col  class="org-right" />
 
-<col  class="left" />
+<col  class="org-left" />
 
-<col  class="left" />
+<col  class="org-left" />
 </colgroup>
 <thead>
 <tr>
-<th scope="col" class="left">Flag Name</th>
-<th scope="col" class="right">Default value</th>
-<th scope="col" class="left">Clang/LLVM Version</th>
-<th scope="col" class="left">Description</th>
+<th scope="col" class="org-left">Flag Name</th>
+<th scope="col" class="org-right">Default value</th>
+<th scope="col" class="org-left">Clang/LLVM Version</th>
+<th scope="col" class="org-left">Description</th>
 </tr>
 </thead>
 
 <tbody>
 <tr>
-<td class="left">flush&#95;shadow</td>
-<td class="right">0</td>
-<td class="left">>= 4.0</td>
-<td class="left">Flush shadow memory at the end of an outer OpenMP parallel region. Experiments show a memory overhead reduction of ~30%, and an increase of the runtime overhead of ~10%. This flag is useful for large OpenMP applications that normally requires a large amount of memory and may cause an "out of memory" exception when checked by Archer.</td>
+<td class="org-left">flush&#95;shadow</td>
+<td class="org-right">0</td>
+<td class="org-left">>= 4.0</td>
+<td class="org-left">Flush shadow memory at the end of an outer OpenMP parallel region. Experiments show a memory overhead reduction of ~30%, and an increase of the runtime overhead of ~10%. This flag is useful for large OpenMP applications that normally requires a large amount of memory and may cause an "out of memory" exception when checked by Archer.</td>
+</tr>
+</tbody>
+
+<tbody>
+<tr>
+<td class="org-left">print&#95;ompt&#95;counters</td>
+<td class="org-right">0</td>
+<td class="org-left">>= 3.9</td>
+<td class="org-left">Print the number of triggered OMPT events at the end of the execution.</td>
+</tr>
+</tbody>
+
+<tbody>
+<tr>
+<td class="org-left">print&#95;max&#95;rss</td>
+<td class="org-right">0</td>
+<td class="org-left">>= 3.9</td>
+<td class="org-left">Print the RSS memory peak at the end of the execution.</td>
 </tr>
 </tbody>
 </table>
 
-# Example<a id="sec-6" name="sec-6"></a>
+
+<a id="org84b0be1"></a>
+
+# Example
 
 Let us take the program below and follow the steps to compile and
 check the program for data races.
 
 Suppose our program is called *myprogram.c*:
 
-     1  #include <stdio.h>
-     2
-     3  #define N 1000
-     4
-     5  int main (int argc, char **argv)
-     6  {
-     7    int a[N];
-     8
-     9  #pragma omp parallel for
-    10    for (int i = 0; i < N - 1; i++) {
-    11      a[i] = a[i + 1];
-    12    }
-    13  }
+    nilbody
 
 In case we installed Archer with the official LLVM OpenMP runtime and
 ThreadSanitizer support we compile the program as follow:
```

Comment:
Add a comma? 

In case we installed Archer with the official LLVM OpenMP runtime and ThreadSanitizer support, we compile the program as follow: